static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.tsx

```diff
@@ -115,22 +166,35 @@ function WidgetPreviewContainer({
               location={location}
               forceTransactions={metricsDataSide.forceTransactionsOnly}
             >
-              <SampleWidgetCard
-                initial={{opacity: 0, x: '50%', y: 0}}
-                animate={{opacity: 1, x: 0, y: 0}}
-                exit={{opacity: 0, x: '50%', y: 0}}
-                transition={{
-                  type: 'spring',
-                  stiffness: 500,
-                  damping: 50,
+              <DraggableWidgetContainer
+                ref={setNodeRef}
+                id={WIDGET_PREVIEW_DRAG_ID}
+                style={{
```

Comment:
We might be able to eek out some small perf gains by telling the browser these values will change.
https://developer.mozilla.org/en-US/docs/Learn/Performance/CSS#optimizing_element_changes_with_will-change

i.e. you can probably add `willChange: 'opacity, transform'`

Reply:
hmmm it seems transform won't change even with it in the `willChange` prop because it's a `motion.div` element booooo. I'll see if there's something i can do with motion.div itself tomorrow.